cube/bmp_8.cpp

```diff
@@ -77,14 +73,14 @@ unsigned char **Bmp8::CreateBmpBuffer(unsigned char init_val) {
 
   buff = (unsigned char **) new unsigned char *[hgt_ * sizeof(*buff)];
   if (!buff) {
```

Comment:
>Can any smart pointers help to improve the source code further?
There is a disscution about thos topic. See https://groups.google.com/forum/#!topic/tesseract-dev/mZ2IUsvWgbY